node/src/actors/chain_manager/mod.rs

```diff
@@ -3528,19 +3593,19 @@ mod tests {
             assert_ne!(block_1, block_mal_1);
 
             // Process the modified candidate first
-            chain_manager.process_candidate(block_mal_1);
+            chain_manager.process_candidate(&mut Context::new(), block_mal_1, (0, 0));
```

Comment:
I didn't know it was possible to call `Context::new()`... But maybe it's better to send an `AddCandidates` message? This is inside an async block so we can just call `send().await`.